README.adoc

```diff
@@ -1,22 +1,177 @@
-
-== Original description
-Ce projet permettra à des enseignants de créer des ensembles de questions et réponses. Ces questions peuvent être réutilisées par d’autres enseignants et combinées pour créer des questionnaires ou des sondages. Les étudiants pourront se servir de ces ensembles de questions pour s’entrainer.
-
-=== Retrouvez notre application en suivant le lien ci dessous
-http://collaborative-exams.eu-gb.mybluemix.net/
-
+= Première itération 
+
+*Amine* 
+
+* Création d'un nouvel auteur en renseignant un formulaire
+* Authentification d'un auteur à l'aide de ses identifiants
+* Création des Servlets associés aux auteurs
+* Création d'une vue permettant l'authentification
+* Création d'une vue permettant l'insertion des données lors de la création d’un nouvel auteur
+* Insérer les nouveaux auteurs dans la BDD
+* Gérer en JSON et XML les auteurs 
+
+
+*Remarque*
+
+
+
+Il s’agit d’une fonctionnalité proposée car à ce jour, cette dernière n’a pas été traitée.
+Ainsi, nous ne faisons pas la différence entre les personnes qui créent des questions (Auteurs) et celles qui utilisent les questions pour s'entraîner.
+
+
+
+
+*Khaled*
+
+
+
+* Niveau de difficulté : Pour les QCM, les réponses proposées auront un niveau de difficulté. Les réponses fausses aurons deux compteurs, un pour compter le nombre de participation aux examens, et l'autre pour compter le nombre de fois qu'elle a été sélectionnée par les candidat. le ratio [nombre participation / nombre de sélection ] donne le degré de corrélation entre cette réponse et la bonne réponse, et donc si le ratio est proche de 1 la réponse aura une forte probabilité d'être choisi. Si cette réponse apparaît dans l'examen, on dit que la question est difficile.
+
+
+
+*Remarque*
+au début toute les réponse sont proposées au même niveau de corrélation, après un certain nombre d'utilisation
+
+*Mohamed*
+
+
+* Question Xml : Lire et écrire une question en XML, et les relations de votre modèle. Chaque relation peut être représentée comme un ensemble de triplets associé à une personne(1)
+
+* Faire les différentes conversion en Xml (QuestionText)
+
+
+* Servlet Delet:   Ajouter une servlet qui permet aux autor de supprimer les questions de leur choix dans la BDD avec un cookie qui mémorise id personne. la suppression se fait soit avec un id questions, suppression total,par un id author... Dans la servlet elle permet aussi au enseignants de supprimer les examens qui seront créés (1)
+
+
+*Valentin*
+
+
+
+* Gestion des questions : Un utilisateur peut indiquer sur chaque question: un coefficient pour chaque question, la matière, le chapitre associé et le niveau recommandé
+Création de regroupement de questions appelé Examen selon la matière, le chapitre ou le niveau recommandé (de CP à Terminale)
+* Les examens seront exportables en fichier .pdf
+* Création de la page Author reprenant les noms des auteurs des questions et divers statistiques

```

Comment:
Trop tôt pour interface graphique. Concentrez-vous d’abord sur des servlets qui marchent, avec tests unitaires.
Vous vous êtes déjà donné beaucoup de travail pour cette itération !

Reply:
Commentaire à prendre en compte.

Reply:
Commentaire à prendre en compte.